repondeur/zam_repondeur/fetch/an/liasse_xml.py

```diff
@@ -175,7 +176,6 @@ def check_same_lecture(
         legislature != str(lecture.texte.legislature)
         or partie != lecture.partie
         or organe != lecture.organe
-        or texte_uid != lecture.texte.uid
```

Comment:
Unsure about that one, is the new check enough to consider it is the same lecture?

Reply:
I can think of two ways we could do it:
- `int(texte_uid[-4:]) == lecture.texte.num` (easiest, but it's better not to depend on the structure of the `uid`
- use the `uid` to get the `TexteRef` from the open data, then compare attributes with `lecture.texte` (right now we don't save the `textes` key in the data repo, but we could add it quite easily I think)